gdx-controllers-desktop/src/main/java/com/badlogic/gdx/controllers/desktop/support/JamepadController.java

```diff
@@ -51,7 +53,7 @@ public boolean getButton(final int buttonCode) {
         try {
             ControllerButton button = toButton(buttonCode);
             return button != null && controllerIndex.isButtonPressed(button);
-        } catch (ControllerUnpluggedException e) {
+        } catch (ControllerUnpluggedException|NullPointerException e) {
```

Comment:
We are not as restrictive as the main repo here, but can you apply a formatter to insert spaces here? :)